protocol/activity_center_persistence.go

```diff
@@ -17,6 +17,44 @@ func (db sqlitePersistence) DeleteActivityCenterNotification(id []byte) error {
 	return err
 }
 
+func (db sqlitePersistence) DeleteActivityCenterNotificationForMessage(chatID string, messageID string) error {
+	var tx *sql.Tx
+	var err error
+
+	tx, err = db.db.BeginTx(context.Background(), &sql.TxOptions{})
+	if err != nil {
+		return err
+	}
+	defer func() {
+		if err == nil {
+			err = tx.Commit()
+			return
+		}
+		// don't shadow original error
+		_ = tx.Rollback()
+	}()
+
+	_, notifications, err := db.buildActivityCenterQuery(tx, "", 0, nil, chatID)
+
+	if err != nil {
+		return err
+	}
+
+	var ids []types.HexBytes
+
+	for _, notification := range notifications {
+		if notification.Message != nil && notification.Message.ID == messageID {
+			ids = append(ids, notification.ID)
+		}
+	}
+
+	if len(ids) > 0 {
+		return db.DismissActivityCenterNotifications(ids)
```

Comment:
When this is executed it _seems_ to work but after that can't retrieve any messages from any chat, app does not load chat messages. Any clue what's happenning? cc @cammellos @Samyoul

Reply:
Nevermind, DB connection was being closed after this DB query execution.